lib/mixlib/install/generator/powershell/scripts/helpers.ps1.erb

```diff
@@ -141,7 +141,7 @@ function Get-SHA256Converter {
     [System.Security.Cryptography.SHA256]::Create()
   }
   else {
-    New-Object -TypeName Security.Cryptography.SHA256Managed
+    New-Object -TypeName Security.Cryptography.SHA256Cng
```

Comment:
we should add a comment here saying "Using SHA256Cng instead of SHA256Managed because SHA256Managed is not FIPS 140-2 compatible"